gcloud-java-storage/src/main/java/com/google/gcloud/storage/Storage.java

```diff
@@ -695,6 +695,10 @@ public static BlobListOption prefix(String prefix) {
 
     /**
      * Returns an option to specify whether blob listing should include subdirectories or not.
+     * {@link StorageOptions#pathDelimiter()} is used as a path delimiter. If set to {@code true}
+     * also blobs in subdirectories are listed. If set to {@code false} and used in combination
```

Comment:
> I am not sure I understand the "and used in combination" part.

I wanted to say that if used in combination `prefix` it enables to list the content of a specific "directory". I agree it's unclear as is.

> Since we're not really talking about "directories" it might be good to be more precise here and avoid using "directories" and "subdirectories" without more context.

I agree but this is not easy to be explained without using the concept of directory and subdirectory. In fact that is how it's explained in the [official docs](https://cloud.google.com/storage/docs/json_api/v1/objects/list).